layouts/auto_generated_product.tpl

```diff
@@ -0,0 +1,125 @@
+<!DOCTYPE html>
+{% include "template-variables" %}
+
+<html class="{% if editmode %}editmode{% else %}public{% endif %}" lang="{{ page.language_code }}">
+<head prefix="og: http://ogp.me/ns#">
+  {% assign items_page = true %}
+  {% include "edicy-tools-variables" %}
+  {% include "html-head" %}
+  {% include "edicy-tools-styles" %}
+</head>
+
+<body class="item-page content-page{% unless editmode or site_header_has_content %} empty-site-header{% endunless %} js-bg-picker-area{% if fallback_state %} bgpicker-fallback{% endif %}">
+  {% include "template-svg-spritesheet" %}
+  {% if editmode %}<button class="bgpicker-btn js-background-settings" data-bg-key="body_bg" data-bg-default-image-color="rgb(111, 108, 119)" data-bg-image="{{ body_bg_image }}" data-bg-image-sizes="{{ body_bg_image_sizes_str | escape }}" data-bg-color="{{ body_bg_color }}" data-bg-color-data="{{ body_bg_color_data_str | escape }}"></button>{% endif %}
+  {% if body_bg_image != '' or editmode %}<div class="body-background-image js-background-image"></div>{% endif %}
+  {% if body_bg_color != '' or editmode %}<div class="body-background-color js-background-color"></div>{% endif %}
```

Comment:
Since `page.data` is unavailable on the auto-generated product page, the bgpicker isn't needed here.

Reply:
Remark. `page.data` **is** available on the auto-generated product page, but it changes the store mount point (root) page bg. So it may be confusing and its really better to remove at the moment.